tests/integration_tests/functional/test_snapshot_basic.py

```diff
@@ -143,15 +150,35 @@ def _test_seq_snapshots(context):
                                                      tapname="tap0")
     logger.debug("Host IP: {}, Guest IP: {}".format(host_ip, guest_ip))
 
+    # Configure vsock device.
+    vsock_uds_path = "v.sock"
+    basevm.vsock.put(
+        vsock_id="vsock0",
+        guest_cid=3,
+        uds_path="/{}".format(vsock_uds_path)
+    )
+    # Generate a random data file for vsock.
+    blob_path, blob_hash = _make_blob(test_session_root_path)
+
     # We will need netmask_len in build_from_snapshot() call later.
     netmask_len = network_config.get_netmask_len()
     basevm.start()
-    ssh_connection = net_tools.SSHConnection(basevm.ssh_config)
 
+    ssh_connection = net_tools.SSHConnection(basevm.ssh_config)
     # Verify if guest can run commands.
     exit_code, _, _ = ssh_connection.execute_command("sync")
     assert exit_code == 0
 
+    # Copy the data file and a vsock helper copy to the guest.
+    cmd = "mkdir -p /tmp/vsock && mount -t tmpfs tmpfs /tmp/vsock"
+    ecode, _, _ = ssh_connection.execute_command(cmd)
+    assert ecode == 0
```

Comment:
Please add an error message after the assert statement so that it's easier to debug potential problems during automated tests.

Reply:
Good catch. Done.